scripts/prepare_alpaca.py

```diff
@@ -54,9 +54,9 @@ def prepare(
     tokenizer = Tokenizer(checkpoint_dir)
 
     # Partition the dataset into train and test
-    train_split_size = len(data) - test_split_size
+    train_split_percent = 100 - test_split_percent
     train_set, test_set = random_split(
-        data, lengths=(train_split_size, test_split_size), generator=torch.Generator().manual_seed(seed)
+        data, [train_split_percent / 100., test_split_percent / 100.], generator=torch.Generator().manual_seed(seed)
```

Comment:
```suggestion
        data, (train_split_percent, test_split_percent), generator=torch.Generator().manual_seed(seed)
```

Reply:
Why a tuple instead of a list?  The examples at https://pytorch.org/docs/stable/data.html are lists.  I suppose either will work, but I prefer to follow any usage shown in the documentation.

Reply:
It's a very minor nit. But if you don't need your sequence to be mutable, it's faster to use tuples